compiler/rustc_passes/src/dead.rs

```diff
@@ -644,32 +689,46 @@ struct DeadVisitor<'tcx> {
     ignored_derived_traits: &'tcx FxHashMap<LocalDefId, Vec<(DefId, DefId)>>,
 }
 
+enum ShouldWarnAboutField {
+    Yes(bool), // positional?
```

Comment:
```suggestion
    Named,
    Positional,
```
This will probably be clearer.